bw_matchbox/webapp.py

```diff
@@ -437,14 +437,24 @@ def search():
 
     q = flask.request.args.get("q")
     embed = flask.request.args.get("e")
+    json = flask.request.args.get("json")  # Return the data as json
     search_db = flask.request.args.get("database")
 
-    if embed:
+    table_data = bd.Database(search_db or t).search(q, limit=100),
```

Comment:
There shouldn't be a trailing comma here, that creates a tuple instead of a list of results.

```python
In [1]: a = "foo",

In [2]: a
Out[2]: ('foo',)
```